projects/angular/src/forms/combobox/combobox.ts

```diff
@@ -82,23 +82,28 @@ export class ClrCombobox<T>
   extends WrappedFormControl<ClrComboboxContainer>
   implements ControlValueAccessor, LoadingListener, AfterContentInit
 {
-  @ViewChild('textboxInput') textbox: ElementRef;
-  @ViewChild('trigger') trigger: ElementRef;
+  /** The native `<input class="clr-input clr-combobox-input" />` element. */
+  @ViewChild('textboxInput', { static: true }) textbox: ElementRef<HTMLInputElement>;
```

Comment:
Why we want to run the query before the change detection?

Reply:
Queries are always run during the change detection:
```ts
if (rf & ɵRenderFlags.Update) {
  let _t;
  ɵɵqueryRefresh((_t = ɵɵloadQuery())) && (ctx.textbox = _t.first);
}
```

I'm not sure if we have to mark static elements (which are not placed within embedded views) as `{ static: false }`.